vcd/resource_vcd_vapp_vm.go

```diff
@@ -668,7 +679,7 @@ func resourceVcdVAppVmUpdate(d *schema.ResourceData, meta interface{}) error {
 }
 
 func resourceVcdVAppVmUpdateExecute(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Update should avoid any code handling if only network_dhcp_wait_seconds is changed. Or may call read instead ^^

Reply:
And it does. There is nothing happening when only this value is changed.

Reply:
You mean update function isn't called?

Reply:
The function is called, because I can't do "DiffSuppressFunc" (for the reason that the value doesn't get to state and Read only gets values from read), but all actual changes in Update are guarded by `d.HasChange`

If I explicitly list all of the fields we have, it will have to be "maintained" everytime and I didn't find a function that would say something like `d.IfOnlyFieldChanged("wait_for_dhcp_seconds")` although that would be good to possibly exit early.

Reply:
It's risk one against another. I mean early(though have to maintain) exist save time(as now it makes at least 3 request before ends function), but makes it secure that nothing will called when someone will add new features and it won't be guarded for some reason.

Reply:
I will check if I can make some function having behavior as `d.IfOnlyFieldChanged("wait_for_dhcp_seconds")` with what is available.

Reply:
I have implemented a custom fb837a5 function to check that based on the schema of resource (didn't find anything useful in `d.XXX` functions)